pkg/skaffold/watch/triggers_test.go

```diff
@@ -51,6 +52,13 @@ func TestNewTrigger(t *testing.T) {
 			opts:        &config.SkaffoldOptions{Trigger: "manual"},
 			expected:    &manualTrigger{},
 		},
+		{
+			description: "api trigger",
+			opts:        &config.SkaffoldOptions{Trigger: "api"},
+			expected: &apiTrigger{
+				Interval: time.Duration(0) * time.Millisecond,
```

Comment:
Can we add another test (or change this one) and set polling interval in opts? 
Also verify the Interval is set to `Opts.WatchPollInterval`.